internal/operands/vm-console-proxy/reconcile_test.go

```diff
@@ -35,12 +40,20 @@ const (
 )
 
 var _ = Describe("VM Console Proxy Operand", func() {
-	var v *vmConsoleProxy
-	var mockedRequest *common.Request
+	const (
+		replicas int32 = 1
+	)
+
+	var (
+		bundle  *vmConsoleProxy
+		operand *vmConsoleProxy
+		request common.Request
+	)
 
 	BeforeEach(func() {
-		v = getMockedVmConsoleProxyOperand()
-		mockedRequest = getMockedRequest()
+		bundle = New(getMockedTestBundle())
+		operand = getMockedVmConsoleProxyOperand()
+		request = getMockedRequest()
 	})
 
 	It("should return new with test bundle correctly", func() {
```

Comment:
Please remove this test, because it only tests private fields. Usually unit tests are only for exported functionality.